### PR TITLE
add code for flipkart price alert

### LINKED ISSUE #541: Flipkart Price Alert
# Aim
I am a participant in GSSOC'21.
As the title says , the script checks for the price of a desired product periodically. If the price drops below the desired amount, notifies the user via email.

# Details
Will be using BeautifulSoup to scrape webpages.

### PR DESCRIPTION
**Pull request by Jade9ja**
# Description

Added the flipkart price alert script.

## Fixes #541 

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [X] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [X] My code follows the style guidelines(Clean Code) of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [X] My changes generate no new warnings
- [X] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Flipkart-price-alert/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Flipkart Price Alert
+Checks for the price of a desired product periodically.
+If the price drops below the desired amount, notifies the user via email.
+## Setup Instructions
+Type the following to install requirements: (make sure you have python installed)
+`pip install -r requirements.txt`
+# How to get app password
```
Comment (kaustubhgupta) [inline Flipkart-price-alert/README.md:7]: Not all of us store passwords in Google account. You can remove this section

File: Flipkart-price-alert/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+![](https://devanswers.co/wp-content/uploads/2017/02/my-google-app-passwords.png)
+
+# Usage
+1) Edit the config vars in the track.py script.
+2) Run the track.py script. `python track.py`
+3) Hope the price drops :).
+
+## Author(s)
+Ea (github.com/jade9ja)
```
Comment (kaustubhgupta) [inline Flipkart-price-alert/README.md:21]:
```suggestion
[Ea](github.com/jade9ja)
```

File: Flipkart-price-alert/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Disclaimers
```
Comment (kaustubhgupta) [inline Flipkart-price-alert/README.md:23]: are you sure this disclaimer has anything to do with this script?

File: Flipkart-price-alert/track.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import requests
+from bs4 import BeautifulSoup as bs
+import smtplib
+import time
+# BEGIN CONFIG
+headers = {'User-Agent': 'Mozilla/5.0 Chrome/86.0.4240.75'}
+# Enter the URL of the product page on flipkart
+PRODUCT_URL = ''
```
Comment (kaustubhgupta) [inline Flipkart-price-alert/track.py:8]: Get all these parameters from the user via terminal, Print statements

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Flipkart-price-alert/README.md (hunk 3 above)
Comment (Jade9ja) [reply Flipkart-price-alert/README.md:23]: The app passwords that one might generate for the script can still be used by others if the user accidentally leaks it hence the disclaimer for the same.

I will change/ remove the second disclaimer.

File: Flipkart-price-alert/README.md (hunk 1 above)
Comment (Jade9ja) [reply Flipkart-price-alert/README.md:7]: the smtp library requires the user's mail credentials, this details the way in which one can get the said credentials but for a limited scope (ie. just for accessing mail)

**Comment by Jade9ja** [comment]
I have changed the stuff that you requested apart from [this](https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/548#discussion_r593842245) (I explained why its needed [here](https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/548#discussion_r593850875)) If it all looks good then shall I close this pr and open a new one?

**Comment by kaustubhgupta** [comment]
@Jade9ja no you didn't need to make a new PR but you already did it

**Pull request closed without merging**